tests/templates/kuttl/spark-history-server/03-setup-minio.yaml

```diff
@@ -25,60 +27,5 @@ spec:
   restartPolicy: Never
   containers:
     - name: minio-client
-      image: docker.io/bitnami/minio-client:2022.8.11-debian-11-r3
-      command: ["bash", "-c", "sleep infinity"]
-      stdin: true
-      tty: true
-      env:
-        - name: MINIO_SERVER_ACCESS_KEY
-          valueFrom:
-            secretKeyRef:
-              name: minio-credentials
-              key: root-user
-              optional: false
-        - name: MINIO_SERVER_SECRET_KEY
-          valueFrom:
-            secretKeyRef:
-              name: minio-credentials
-              key: root-password
-              optional: false
-        - name: MINIO_SERVER_HOST
-          value: test-minio
-        - name: MINIO_SERVER_PORT_NUMBER
-          value: "9000"
-        - name: MINIO_SERVER_SCHEME
-          value: http
----
-apiVersion: v1
-kind: Pod
-metadata:
-  name: eventlog-minio-client
-  labels:
-    app: eventlog-minio-client
-spec:
-  restartPolicy: Never
-  containers:
-    - name: minio-client
-      image: docker.io/bitnami/minio-client:2022.8.11-debian-11-r3
+      image: docker.io/bitnami/minio-client:2023.5.18-debian-11-r2
```

Comment:
I think we can get rid of this pod and use the minio (server) directly. That should already be set up properly.

Reply:
As discussed - will leave this as is.